modules/aggregation-layers/src/utils/gpu-grid-aggregation/grid-aggregation-utils.js

```diff
@@ -108,15 +108,18 @@ function parseGridData(data, getPosition, getWeight = null) {
     positions64xyLow[index * 2] = fp64LowPart(x);
     positions64xyLow[index * 2 + 1] = fp64LowPart(y);
 
-    const weight = getWeight ? getWeight(object, objectInfo) : DEFAULT_WEIGHT;
-    // Aggregator expects each weight is an array of size 3
-    if (Array.isArray(weight)) {
-      weightValues[index * 3] = weight[0];
-      weightValues[index * 3 + 1] = weight[1];
-      weightValues[index * 3 + 2] = weight[2];
-    } else {
-      // backward compitability
-      weightValues[index * 3] = weight;
+    for (const name in weightParams) {
+      const weight = weightParams[name].getWeight(object);
```

Comment:
Is this a breaking change?

Reply:
this is a private API used internally by layers within `aggregation-layers` module, I just double checked and removed an export (was an experimental) from `aggregation-layers`.